e2e/libs/volume/rest.py

```
@@ -217,6 +217,23 @@ def get_replica_name_on_node(self, volume_name, node_name):
+            running_replica_count = 0
+            volume = get_longhorn_client().by_id_volume(volume_name)
+            for r in volume.replicas:
+                if node_name and r.hostId == node_name and r.running:
+                    running_replica_count += 1
+                elif not node_name and r.running:
+                    running_replica_count += 1
+            logging(f"Waiting for {replica_count if replica_count else ''} replicas for volume {volume_name} running on {node_name if node_name else 'nodes'}, currently it's {running_replica_count} ... ({i})")
+            if replica_count and running_replica_count == int(replica_count):
+                break
+            elif not replica_count and running_replica_count:
+                break
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)